tldr.py

```diff
@@ -277,14 +276,22 @@ def main():
 
     options = parser.parse_args(other_options)
 
-    for command in options.command:
-        try:
-            if options.os is not None:
-                output(get_page(command, options.os))
-            else:
-                output(get_page(command))
-        except Exception:
-            print("No internet connection detected. Please reconnect and try again.")
+    try:
+        result = get_page('-'.join(options.command), options.os)
+        if not result:
+            for command in options.command:
```

Comment:
You are not checking for `options.os is not None`. Is that not necessary anymore ?

Reply:
Yes. `get_page(command)` is same as `get_page(command, None)`, because default value for `platform` is None ([tldr.py#L153](https://github.com/iliakonnov/tldr-python-client/blob/3891ba4482732d0f835aac4e7215467e56a4c188/tldr.py#L153))